spanner/spanner_snippets/snippet.go

```diff
@@ -26,20 +26,28 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/genproto/googleapis/longrunning"
+	"google.golang.org/grpc/codes"
+
 	"cloud.google.com/go/civil"
 	"cloud.google.com/go/spanner"
-	database "cloud.google.com/go/spanner/admin/database/apiv1"
 	"google.golang.org/api/iterator"
 	"google.golang.org/api/option"
 	"google.golang.org/grpc"
 
+	spadmindb "cloud.google.com/go/spanner/admin/database/apiv1"
```

Comment:
Why rename this to a different package name?

Reply:
There is a function parameter is also named as `database`. So I can't refer to a protobuf object inside the function. E.g., previously, 

```go
func listBackups(ctx context.Context, w io.Writer, adminClient *database.DatabaseAdminClient, database, backupID string) {
        // This throws an error saying `string` doesn't have a field "BackupIterator"
    	printBackups := func(iter *database.BackupIterator) error {
}
```

If you think this change it too large, I can just change the function parameter `database string` to `db string`.

Reply:
Yeah unfortunately we have to avoid making changes that affect all the samples because it means we have to regenerate all the Cloud docs.

Reply:
Reverted back.